src/components/download-item/download-item.scss

```diff
@@ -1,7 +1,8 @@
+@import '~@playkit-js/playkit-js-ui';
 @import '../../theme.scss';
 
 .file-info {
-  background-color: rgba(255, 255, 255, 0.12);
+  background-color: rgba(#{$tone-1-color-rgb}, 0.1);
```

Comment:
so this means that any change in tone 1 will affect download plugin now. is that what product want ?

Reply:
Exactly, note` 255, 255, 255` is the representation of tone-1 but hard-coded, so it is not effected on light theme